pydemolib/tests/test_with_iceberg_flink.py

```diff
@@ -0,0 +1,171 @@
+            .field("BLK", DataTypes.STRING())
+            .field("TOV", DataTypes.STRING())
+            .field("PTS", DataTypes.STRING())
+            .field("Player", DataTypes.STRING())
+            .field("RSorPO", DataTypes.STRING())
+        ).with_schema(
+            Schema()
+            .field("Season", DataTypes.STRING())
+            .field("Age", DataTypes.STRING())
+            .field("Team", DataTypes.STRING())
+            .field("ORB", DataTypes.STRING())
+            .field("DRB", DataTypes.STRING())
+            .field("TRB", DataTypes.STRING())
+            .field("AST", DataTypes.STRING())
+            .field("STL", DataTypes.STRING())
+            .field("BLK", DataTypes.STRING())
+            .field("TOV", DataTypes.STRING())
+            .field("PTS", DataTypes.STRING())
+            .field("Player", DataTypes.STRING())
+            .field("RSorPO", DataTypes.STRING())
+        ).create_temporary_table(
+            f"{catalog_name}.nba.totals_stats_temp"
+        )
+
+        dev_table_env.execute_sql(
+            f"CREATE TABLE IF NOT EXISTS {catalog_name}.nba.totals_stats (Season STRING, Age STRING, Team STRING, \
+            ORB STRING, DRB STRING, TRB STRING, AST STRING, STL STRING, BLK STRING, TOV STRING, PTS STRING, \
+            Player STRING, RSorPO STRING)"
+        ).wait()
+
+        tab = dev_table_env.from_path(f"{catalog_name}.nba.totals_stats_temp")
+        tab.select(
+            tab.Season, tab.Age, tab.Team, tab.ORB, tab.DRB, tab.TRB, tab.AST, tab.STL, tab.BLK, tab.TOV, tab.PTS, tab.Player, tab.RSorPO
+        ).execute_insert(f"{catalog_name}.nba.totals_stats").wait()
+
+        from pyflink.table.expressions import lit
+
+        num_salaries = table_env.from_path(f"{catalog_name}.nba.`salaries@dev`").select(lit(1).count).to_pandas().to_string()
+
+        assert_that(num_salaries).contains("51")
+        assert_that(num_salaries).is_equal_to(
+            dev_table_env.from_path(f"{catalog_name}.nba.salaries").select(lit(1).count).to_pandas().to_string()
+        )
+
+        # verify that the table does not exist on the main branch
+        expect_error("py4j.protocol.Py4JJavaError", lambda: table_env.from_path(f"{catalog_name}.nba.salaries"))
+        expect_error("py4j.protocol.Py4JJavaError", lambda: table_env.from_path(f"{catalog_name}.nba.`salaries@main`"))
+
+        run(["nessie", "merge", "dev", "-b", "main", "--force"])  # noqa: S603 S607
+
+        assert_that(table_env.from_path(f"{catalog_name}.nba.`salaries@main`").select(lit(1).count).to_pandas().to_string()).is_equal_to(
+            num_salaries
+        )
+
+    @pytest.mark.skip("Skipped until we get a new Iceberg release")
+    @pytest.mark.forked
+    def test_with_iceberg_flink(self: object) -> None:
+        """Test NessieDemo+Iceberg+Flink against Nessie 0.6.x + Iceberg 0.11.x."""
+        TestWithIcebergFlink.__test_with_iceberg_flink("nessie-0.6-iceberg-flink-0.11.yml", [])
```

Comment:
Can you move this into `configs/in-development/` ?

Reply:
yep will do a follow-up PR shortly to fix this